embark-anyone.lua

```diff
@@ -0,0 +1,108 @@
+local dialogs = require 'gui.dialogs'
+local choices = {}
+
+function addCivToEmbarkList(info)
+   local viewscreen = dfhack.gui.getDFViewscreen(true)
+
+   viewscreen.start_civ:insert ('#', info.civ)
+   viewscreen.start_civ_nem_num:insert ('#', info.nemeses)
+   viewscreen.start_civ_entpop_num:insert ('#', info.pops)
+   viewscreen.start_civ_site_num:insert ('#', info.sites)
+end
+
+function embarkAnyone()
+   local viewscreen = dfhack.gui.getDFViewscreen(true)
+
+
+   if viewscreen._type ~= df.viewscreen_choose_start_sitest then
+      qerror("This script can only be used on the embark screen!")
+   end
+
+   for i, civ in ipairs (df.global.world.entities.all) do
+      if civ.type == df.historical_entity_type.Civilization then
+         local available = false
+
+         -- Check if civ is already available to embark
+         for _, item in ipairs (viewscreen.start_civ) do
+            if item == civ then
+               available = true
+               break
+            end
+         end
```

Comment:
this would be more efficient if you built a set outside of this loop. e.g.:
```lua
local existing_civs = {}
for _,existing_civ in ipairs(viewscreen.start_civ) do
    existing_civs[existing_civ.id] = true
end
```
Then later you can test for membership by evaluating `existing_civs[civ.id]`:
```lua
if existing_civs[civ.id] then goto continue end
```